irohad/ordering/impl/on_demand_ordering_service_impl.cpp

```diff
@@ -97,9 +90,8 @@ OnDemandOrderingServiceImpl::onRequestProposal(consensus::Round round) {
   // read lock
   std::shared_lock<std::shared_timed_mutex> guard(lock_);
   auto proposal = proposal_map_.find(round);
-  log_->debug("onRequestProposal, round[{}, {}], {}returning a proposal.",
-              round.block_round,
-              round.reject_round,
+  log_->debug("onRequestProposal, {}, {}returning a proposal.",
```

Comment:
```suggestion
  log_->debug("onRequestProposal, {}, {} returning a proposal.",
```

Reply:
https://github.com/hyperledger/iroha/pull/1871#discussion_r238148963

Reply:
:laughing: maybe we need to place a comment warning that the whitespace is not missing there! To see why, please see the substitution for these braces.